pnpm-lock.yaml

```diff
@@ -5,7 +5,6 @@ settings:
   excludeLinksFromLockfile: false
 
 overrides:
-  ethereumjs-abi: https://registry.npmjs.org/ethereumjs-abi/-/ethereumjs-abi-0.6.8.tgz
```

Comment:
@iPaulPro do you recall any change that could have removed this? It's still present in the `package.json` so unclear to me why it gets removed from the lock file.

Reply:
The lock file was very out of date and this is just removing an override, not the library. If you run `pnpm install` on the current `develop` branch the same change is made.